src/entity/Entity.php

```diff
@@ -1615,6 +1623,7 @@ public function broadcastAnimation(Animation $animation, ?array $targets = null)
 
 	/**
 	 * Broadcasts a sound caused by the entity. If the entity is considered "silent", the sound will be dropped.
+	 *
```

Comment:
here too

Reply:
I have set phpstorm to use the PM config that dylan sent me, no idea why this editorConfig does this. Maybe PM now does it this way?

Reply:
There's a code style provided in the repo itself called `Project`, you can use that. But even so, the styling isn't perfect.

Reply:
I am using Project + editorconfig, it still appears to reformat phpdocs though. Might be a missing setting.